p2p/discovery.py

```diff
@@ -993,21 +1176,21 @@ def _test() -> None:
     else:
         bootstrap_nodes = tuple(
             kademlia.Node.from_uri(enode) for enode in constants.ROPSTEN_BOOTNODES)
-    discovery = DiscoveryProtocol(privkey, addr, bootstrap_nodes, CancelToken("discovery"))
-    loop.run_until_complete(
-        loop.create_datagram_endpoint(lambda: discovery, local_addr=('0.0.0.0', listen_port)))
+
+    if args.v5:
+        # topic = b'LES2@41941023680923e0'  # LES2/ropsten
+        topic = b'LES2@d4e56740f876aef8'  # LES2/mainnet
```

Comment:
Maybe move the strings themselves to a `constants.py` file somewhere?

Reply:
This is in the `_test()` function. In actual production code the topics will be generated from the supported protocols and the hash of the genesis block

Reply:
What are the challenges for moving all the p2p `_test()` methods somewhere into the `tests` folder? Is it just a straight cut/paste job into a `def test_...` method, or is there something tougher?

Reply:
This is not an automated test -- it's meant to be executed manually as it will connect to remote nodes, like the one in, say, `trinity/sync/__main__.py`